src/app/constants/index.tsx

```diff
@@ -0,0 +1,254 @@
+interface BulletPoint {
+  text: string;
+}
+
+interface About {
+  name: string;
+  title: string;
+  skills: string[];
+}
+
+interface Contact {
+  email: string;
+  github: string;
+  linkedin: string;
```

Comment:
so incredibly nitpicky but surely camelcase consistency
```suggestion
  linkedIn: string;
```